readme.md

```diff
@@ -27,15 +27,15 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://github.com/DenysSynenko/layout_moyo-header/)
+- [TEST REPORT LINK](https://github.com/DenysSynenko/layout_moyo-header/report/html_report/)
```

Comment:
Fix the demo and test report link and update it in the PR description 
<img width="269" alt="image" src="https://github.com/user-attachments/assets/54bc9d1c-1aed-4e45-8883-9be6ee081977">
```suggestion
- [DEMO LINK](https://DenysSynenk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DenysSynenk.github.io/layout_moyo-header/report/html_report/)
```

Reply:
It is not working in any way, I adding my user name there but still not works